docs/package.json

```diff
@@ -4,12 +4,12 @@
   "description": "preact material component's site",
   "scripts": {
     "start": "if-env NODE_ENV=production && npm run -s serve || npm run -s dev",
-    "build": "preact build --no-prerender",
-    "build:prod": "preact build --template template.html --no-prerender && npm run copyAssets",
+    "build": "preact build",
+    "build:prod": "preact build --template template.html && npm run copyAssets",
     "copyAssets": "cp -R images *.png *.json build/",
```

Comment:
Would be nice to replace `cp` with something that would work on Windows too. Maybe `cp-cli`? Seems pretty old and not that popular though.

Reply:
this one's a big change, plz open a different issue. will take this up later

Reply:
Sure. I actually changed it in my tests (#626) because I only have a Windows computer on hand right now.